frontend/docs/theme.config.tsx

```diff
@@ -7,9 +7,13 @@ const config = {
     <Image src="/hatchet_logo.png" alt="Hatchet logo" width={120} height={35} />
   ),
   head: () => {
+    const { title } = useConfig()
+
+    const fallbackTitle = "Hatchet Docs"
```

Comment:
nit: do you think `Documentation` would be more Google-friendly than `docs` for SEO purposes? I'm not sure 🤷

Reply:
addressed

Reply:
sounds good! this was a question fwiw, I'm really not sure, but I think maybe long words are good 😅

Reply:
🤷